src/DynamoCore/DynamoCore.csproj

```diff
@@ -315,11 +325,13 @@ IF NOT EXIST "$(OutputPath)\dll" "$(OutputPath)\InstallASMForDynamo.exe" /SILENT
 copy /Y "$(SolutionDir)..\extern\LibG\x64\Release\*" "$(OutputPath)\dll"
 </PostBuildEvent>
     <PostBuildEvent>xcopy /Y "$(SolutionDir)..\extern\ProtoGeometry\*" "$(OutputPath)"
-xcopy  /Y "$(SolutionDir)..\doc\distrib\License.rtf" "$(OutputPath)"
+xcopy /Y "$(SolutionDir)..\extern\ProtoGeometry\ProtoGeometry.xml" "$(OutputPath)\$(UICulture)"
```

Comment:
Why this is an addition here? Is this from the work that @holyjewsus has been doing?